corehq/apps/case_importer/tasks.py

```diff
@@ -78,6 +79,7 @@ def do_import(spreadsheet, config, domain, task=None, chunksize=CASEBLOCK_CHUNKS
     name_cache = {}
     caseblocks = []
     ids_seen = set()
+    track_load = case_load_counter("case_importer", domain)
```

Comment:
case imports update cases by submitting forms. Does this mean that we're double counting here?

Reply:
Interesting, I hadn't noticed that before. It looks like every case is being loaded from the database once during the initial part of the import, and then again during the form submission. As far as I know there is no caching between the two lookups so every case will be fetched from the database at least twice during this process. I think it is reasonable to track load here and in form submissions. Additionally, without this I don't know how we would know how many cases are touched by case imports, which seems like a useful thing to know. What do you think?

Reply:
Didn't remove it, but moved load tracking near lookups in 4a44a12bfcacd3973563cf7665f2d42fb1bb9f32